openvpn-install.sh

```diff
@@ -104,8 +104,8 @@ function installUnbound () {
 			apt-get install -y unbound
 
 			# Configuration
-			echo 'interface: 10.8.0.1
-access-control: 10.8.0.1/24 allow
+			echo 'interface: $VPN_SUBNET.1
```

Comment:
Expressions don't expand in single quotes, use double quotes for that.

```suggestion
			echo "interface: $VPN_SUBNET.1
```